src/analysis/calculus/darboux.lean

```diff
@@ -18,7 +18,8 @@ open_locale topological_space classical
 
 variables {a b : ℝ} {f f' : ℝ → ℝ}
 
-/-- Darboux's theorem: if `a ≤ b` and `f' a < m < f' b`, then `f' c = m` for some `c ∈ [a, b]`. -/
+/-- **Darboux's theorem**: if `a ≤ b` and `f' a < m < f' b`, then `f' c = m` for some 
```

Comment:
There are a few more versions of Darboux's theorem below; did you want to bold those names while you were at it?

Reply:
OK I did that